api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -82,8 +82,8 @@ message AccessListOwner {
   // owner.
   string description = 2;
 
-  // ineligible_status describes if this owner is eligible or not
-  // and if not, describes how they're lacking eligibility.
+  // describes if this owner is eligible or not and if not, describes how
```

Comment:
```suggestion
  // `ineligible_status` describes if this owner is eligible or not and if not, describes how
```
It looks like we're removing the field name in some of these changes. Is this intentional?